release-notes/current.yaml

```diff
@@ -2,7 +2,7 @@ date: Pending
 
 # Changes that are expected to cause an incompatibility with previous versions, such as deletions or modifications to existing APIs.
 breaking changes: |
-  Add a breaking change here
+  The Container `ports` field of the gateway instance has been removed, which will cause the gateway Pod to be rebuilt when upgrading the version.
```

Comment:
do you mean when updating the deployment image ? isnt that already the case today?

Reply:
No, it's because the `ports` are deleted. If upgraded, the deployment's pod template will be recalculated, which will cause the deployment's associated pods to be rebuilt.

Reply:
what upgrade are you referring to, upgrading Envoy Gateway version ?

Reply:
Yes, update the image of `envoy-gateway` deployment.

Reply:
yeah thats fine